src/components/select-field/__tests__/BaseSelectField.test.js

```diff
@@ -578,6 +622,17 @@ describe('components/select-field/BaseSelectField', () => {
         });
     });
 
+    describe('handleClearClick', () => {
+        test('should call handleChange with empty array', () => {
+            const wrapper = shallowRenderSelectField();
+            const instance = wrapper.instance();
+
+            sandbox.mock(instance).expects('handleChange');
```

Comment:
Don't you need `.withArgs` with empty array?

Reply:
Yep, will add